packages/components/src/menu-items-choice/index.tsx

```diff
@@ -58,6 +58,7 @@ function MenuItemsChoice( {
 					<MenuItem
 						key={ item.value }
 						role="menuitemradio"
+						disabled={ item.disabled }
```

Comment:
Given that this adds another possible prop to the allowed choices, this likely needs to be updated in the component README here:

https://github.com/WordPress/gutenberg/blob/138d5f1456fcf2c95435178e1a13058e97940861/packages/components/src/menu-items-choice/README.md#L64

Reply:
Included a mention in `menu-item/README.md`: 7cddb1761545443e6ea5a9ec9c2f2b6392e2e6cc